diff -urN gnome-print-0.26.org/installer/Makefile.am gnome-print-0.26/installer/Makefile.am
--- gnome-print-0.26.org/installer/Makefile.am	Wed Mar  7 14:19:56 2001
+++ gnome-print-0.26/installer/Makefile.am	Wed Mar  7 14:20:18 2001
@@ -24,5 +24,3 @@
 	$(LIBGPA_TEST_LIBS) \
 	$(INTLLIBS)
 
-install-data-local: gnome-font-install
-	$(PERL) $(top_srcdir)/run-gnome-font-install ./gnome-font-install $(datadir) $(top_srcdir)
