diff -ur gnome-print-0.4.orig/Makefile.am gnome-print-0.4/Makefile.am
--- gnome-print-0.4.orig/Makefile.am	Fri Jul  9 07:44:06 1999
+++ gnome-print-0.4/Makefile.am	Sat Jul 17 18:21:18 1999
@@ -78,16 +78,16 @@
 	$(GNOME_XML_LIB)
 
 install-data-local: gnome-font-install
-	./gnome-font-install --system --scan --no-copy 			\
-		--afm-path=$(datadir)/fonts/afms 			\
-		--pfb-path=$(datadir)/fonts/pfbs 			\
-		$(GHOSTSCRIPT_ASSIGN)					\
-		--pfb-assignment=ghostscript,/usr/share/ghostscript/fonts \
-		--pfb-assignment=ghostscript,/usr/lib/ghostscript/fonts   \
-		--pfb-assignment=ghostscript,/usr/share/fonts/default/Type1 \
-		--pfb-assignment=ghostscript,/usr/share/fonts/default/ghostscript \
-		--afm-assignment=ghostscript,/usr/share/ghostscript/fonts \
-		--afm-assignment=ghostscript,/usr/lib/ghostscript/fonts   \
-		--afm-assignment=ghostscript,/usr/share/fonts/default/Type1 \
-		--afm-assignment=ghostscript,/usr/share/fonts/default/ghostscript \
-		$(srcdir)/fonts/
+#	./gnome-font-install --system --scan --no-copy 			\
+#		--afm-path=$(datadir)/fonts/afms 			\
+#		--pfb-path=$(datadir)/fonts/pfbs 			\
+#		$(GHOSTSCRIPT_ASSIGN)					\
+#		--pfb-assignment=ghostscript,/usr/share/ghostscript/fonts \
+#		--pfb-assignment=ghostscript,/usr/lib/ghostscript/fonts   \
+#		--pfb-assignment=ghostscript,/usr/share/fonts/default/Type1 \
+#		--pfb-assignment=ghostscript,/usr/share/fonts/default/ghostscript \
+#		--afm-assignment=ghostscript,/usr/share/ghostscript/fonts \
+#		--afm-assignment=ghostscript,/usr/lib/ghostscript/fonts   \
+#		--afm-assignment=ghostscript,/usr/share/fonts/default/Type1 \
+#		--afm-assignment=ghostscript,/usr/share/fonts/default/ghostscript \
+#		$(srcdir)/fonts/
