diff -Nur gnome-print-0.30-orig/installer/gf-pfb.c gnome-print-0.30/installer/gf-pfb.c
--- gnome-print-0.30-orig/installer/gf-pfb.c	Thu Oct 11 12:15:09 2001
+++ gnome-print-0.30/installer/gf-pfb.c	Thu Oct 11 12:15:27 2001
@@ -43,7 +43,7 @@
 
 	close (fh);
 
-	if (buf == (gpointer) -1) return NULL;
+	if (buf == (gpointer) 0) return NULL;
 
 	if ((buf[0] != 0x80) && (buf[1] != 0x01)) {
 		munmap (buf, s.st_size);
