diff -Nru gnome-print-0.10/Makefile.am gnome-print-0.10.new/Makefile.am
--- gnome-print-0.10/Makefile.am	Mon Oct 18 22:51:32 1999
+++ gnome-print-0.10.new/Makefile.am	Sun Feb 20 06:37:22 2000
@@ -90,19 +90,3 @@
 	$(GNOME_LIBS)		\
 	$(GNOME_XML_LIB)	\
 	$(INTLLIBS)
-
-install-data-local: gnome-font-install
-	./gnome-font-install --system --scan --no-copy 			\
-		--afm-path=$(datadir)/fonts/afms 			\
-		--pfb-path=$(datadir)/fonts/pfbs 			\
-		--fontmap-path=$(datadir)/fonts				\
-		$(GHOSTSCRIPT_ASSIGN)					\
-		--pfb-assignment=ghostscript,/usr/share/ghostscript/fonts \
-		--pfb-assignment=ghostscript,/usr/lib/ghostscript/fonts   \
-		--pfb-assignment=ghostscript,/usr/share/fonts/default/Type1 \
-		--pfb-assignment=ghostscript,/usr/share/fonts/default/ghostscript \
-		--afm-assignment=ghostscript,/usr/share/ghostscript/fonts \
-		--afm-assignment=ghostscript,/usr/lib/ghostscript/fonts   \
-		--afm-assignment=ghostscript,/usr/share/fonts/default/Type1 \
-		--afm-assignment=ghostscript,/usr/share/fonts/default/ghostscript \
-		$(srcdir)/fonts/
