diff -urN gnome-print-0.28.org/configure.in gnome-print-0.28/configure.in
--- gnome-print-0.28.org/configure.in	Mon Apr 23 12:19:32 2001
+++ gnome-print-0.28/configure.in	Mon Apr 23 12:20:18 2001
@@ -62,7 +62,7 @@
 
 GNOME_COMPILE_WARNINGS
 GNOME_X_CHECKS
-AM_GNOME_GETTEXT
+AM_GNU_GETTEXT
 AM_PATH_GDK_PIXBUF(0.7.0, , AC_MSG_ERROR([Need gdk-pixbuf-0.7.0 or later!]))
 
 dnl ==========================================
