--- gnome-print-0.24/libgnomeprint/gnome-font-face.c~	Fri Sep 22 09:23:17 2000
+++ gnome-print-0.24/libgnomeprint/gnome-font-face.c	Fri Nov 10 15:12:35 2000
@@ -966,7 +966,7 @@
 	} else {
 		/* Check gnome-print installation prefix */
 
-		fontmap_fn = g_strconcat(DATADIR, "/fonts/fontmap", NULL);
+		fontmap_fn = "/usr/share/fonts/fontmap";
 
 		if (g_file_exists (fontmap_fn)) gnome_font_load_fontmap (fontmap_fn);
 		g_free (fontmap_fn);
